mflowgen/common/init-fullchip/outputs/gen-bumps.tcl

```diff
@@ -116,7 +116,7 @@ proc gen_bumps {} {
 	"osbssrr444555555666rrssbso"
 	"gsssgsssssbssssbsssssgsssg"
 	"sssgssssgssssssssgssssgsss"
-	"sgossbssssoobboosssgbssogs"
+	"sgossbgsssoobboosssgbssogs"
```

Comment:
So...the seventh bump from the left changed from s to g? Does mean the taped-out chip has a short??

Reply:
Yes, this issue was surfaced in the chip bringup meeting a few weeks ago. Basically, we got lucky that the bad connection was one of the TLX data pins (and not a control pin), so we think the only penalty we'll have to pay for this is reduced TLX bandwidth.